programs/lib/flexpret_exceptions.c

```diff
@@ -4,16 +4,18 @@
 #include "flexpret_assert.h"
 
 #include <flexpret.h>
+#include <errno.h>
 
 typedef void (*isr_t)(void);
 
 static isr_t ext_int_handler;
 static isr_t ie_int_handler;
 static isr_t ee_int_handler;
 
+struct thread_ctx_t contexts[NUM_THREADS];
```

Comment:
I think you should add a little comment here telling what this variable is. Should it also be made static?

Reply:
Making it static gives compilation errors, since the variable is referenced in the ctx_switch.S assembly code, I assume. So I guess it has to be global